_translator-files/po/de/Running-a-Server.po

```diff
@@ -45,123 +47,125 @@ msgid ""
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:26
 msgid "## Basic requirements"
-msgstr ""
+msgstr "## Grundlegende Anforderungen"
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:28
 msgid "While setting up a Server isn't difficult, it's a good idea to read the following background information to avoid some problems:"
-msgstr ""
+msgstr "Die Einrichtung eines Servers ist zwar nicht schwierig, aber es ist ratsam, die folgenden Hintergrundinformationen zu lesen, um einige Probleme zu vermeiden:"
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:30
 msgid "### Speed and latency"
-msgstr ""
+msgstr "### Geschwindigkeit und Latenz"
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:32
 #, no-wrap
 msgid "**_The capability of the Server itself (and the network it's on) is NOT the main determinant of the quality of a Jamulus session!_**\n"
-msgstr ""
+msgstr "**Die Leistungsfähigkeit des Servers selbst (und des Netzes, in dem er sich befindet) ist NICHT ausschlaggebend für die Qualität einer Jamulus-Sitzung!\n"
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:34
 msgid "Usually, problems are on the _Client_ side and should be fixed there. Have a look at the [Troubleshooting page](/wiki/Client-Troubleshooting) if needed."
-msgstr ""
+msgstr "Normalerweise liegen die Probleme auf der _Client_-Seite und sollten dort behoben werden. Wirf einen Blick auf die [Fehlersuche-Seite](/wiki/Client-Troubleshooting)."
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:36
 msgid "However, various problems can also arise when setting up Servers - especially when run on a low-bandwidth home connection. It's usually fine to have less than 5 players on a slower-speed home connection (eg 10 Mbit/s down and 1 Mbit/s up). You can read more about network requirements at [different quality settings here](Server-Bandwidth)."
-msgstr ""
+msgstr "Bei der Einrichtung von Servern können jedoch auch verschiedene Probleme auftreten, insbesondere wenn sie über einen Heimanschluss mit geringer Bandbreite betrieben werden. Normalerweise ist das in Ordnung wenn weniger als 5 Teilnehmer verbunden sind. (z.B. 10 Mbit/s down und 1 Mbit/s up). Weitere Informationen zu den Netzwerkanforderungen findest du unter [verschiedene Qualitätseinstellungen hier](Server-Bandbreite)."
```

Comment:
"Server-Bandwith" at the end should not be translated.

Reply:
Just to be sure:
Should i correct my local file and commit - push the corrected file
Or should I download / sync from branch?

Reply:
Correct it locally and commit I suppose - same with all corrections

Reply:
Thanks - will review all translated files and apply the learnings and 
* Language Code
* ...

If you agree: you dont have to review all pushed .Po files

Reply:
Corrected